### PR TITLE
Ports Eris/Yogstation slime sprites

### PR DESCRIPTION
**Pull request by Erol509**
## About The Pull Request

https://github.com/yogstation13/Yogstation/pull/16942

https://github.com/discordia-space/CEV-Eris/pull/6205

image
image
image

## Why It's Good For The Game

Old slimes are old, I belive they might never actually see any sprite reword from TG/Skyrat side. Cev Eris has nice sprice for them, they are brighter and they are more cute looking now.

## Changelog
:cl:
image: Ported slime sprites from Yog/Cev Eris. Slimes are now prettier.
/:cl:

### CODE REVIEW

**Comment by Elli-Skala** [comment]
> Old slimes are old, I belive they might never actually see any sprite reword from TG/Skyrat side.

They were changed this year.

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Place the slimes into their new modular file and override the icon on the mob

File: modular_zubbers/code/modules/mob/living/simple_animal/slime/slime_sprites_override.dm [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// slimes section
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/mob/living/simple_animal/slime/slime_sprites_override.dm:1]:
```suggestion
// basic slime sprite overrides
```

File: modular_zubbers/code/modules/mob/living/simple_animal/slime/slime_sprites_override.dm [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	icon = 'modular_zubbers/icons/mob/simple/slimes.dmi'
+/obj/item/slime_extract
+	icon = 'modular_zubbers/icons/mob/simple/slimes.dmi'
+/obj/item/stock_parts/cell/high/slime_hypercharged
+	icon = 'modular_zubbers/icons/mob/simple/slimes.dmi'
+
+// slime crossbreeding section
+
+
+/obj/item/slimecross
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/mob/living/simple_animal/slime/slime_sprites_override.dm:16]:
```suggestion
// slime crossbreeding sprite overrides
/obj/item/slimecross
```

**Review by Majkl-J — APPROVED** [review]
I'm not one to exactly rate the sprites, but the code is solid apart from minor comment stuff

**Comment by StrangeWeirdKitten** [comment]
I tested the slimes on the live server and we've come to a conclusion that we will not replace the sprites. 

While I like them, the yogstation / eris slimes are too bright and colorful. They mesh with our asthetic.

**Pull request closed without merging**